src/analysis/unmodified.d

```diff
@@ -198,7 +198,7 @@ private:
 		auto vi = VariableInfo(name);
 		if (guaranteeUse == 0)
 		{
-			const r = tree[index].equalRange(&vi);
+			auto r = tree[index].equalRange(&vi);
```

Comment:
I'll submit a Phobos PR for that (I already have the fix locally, but I want to add some tests), but that's the right way to go, since we support multiple versions of Phobos anyway.